pallets/gear/src/weights.rs

```diff
@@ -56,6 +57,13 @@ pub trait WeightInfo {
 
 pub struct GearWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for GearWeight<T> {
+    fn submit_code(c: u32) -> Weight {
+        (17_289_000_u64)
+            .saturating_add((1_000_u64).saturating_mul(c as Weight))
+            .saturating_add(T::DbWeight::get().reads(1_u64))
+            .saturating_add(T::DbWeight::get().writes(1_u64))
```

Comment:
I don't like the visual noise where it is not obligatory. But we have a lot of such kinds of code, so you can ignore this comment :)

```suggestion
            .saturating_add(T::DbWeight::get().reads(1))
            .saturating_add(T::DbWeight::get().writes(1))
```